consensus/src/detail/tests/indices_tests.rs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2021 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://spdx.org/licenses/MIT
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Author(s): S. Afach
+
+use crate::detail::tests::*;
+
+#[test]
+fn test_indices_calculations() {
```

Comment:
This test should probably be moved to where `calculate_tx_index_from_block()` is

Reply:
So, then we should make importable functions `generate_random_invalid_block` \ `generate_random_invalid_transaction` \ etc.

Reply:
Fixed!